Core/Animation/Playback.cs

```diff
@@ -93,12 +93,12 @@ public virtual void Update(bool idleMotionEnabled = false)
 
         public double BarsFromSeconds(double secs)
         {
-            return secs * Bpm / 240f;
+            return secs * Bpm / 240.0;
         }
         
         public double SecondsFromBars(double bars)
         {
-            return bars * 240 / Bpm;
+            return bars * 240.0 / Bpm;
```

Comment:
Good catch! Although Bpm should be in double, this makes the code much more readable.